app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt

```diff
@@ -67,17 +62,17 @@ class FileRepository(private val context: Context) {
             FileOutputStream(File(pathName))
         }
 
-    fun takeScreenShot(bitmap: Bitmap): File? {
-        val date = Date()
-        val format = DateFormat.format("MM-dd-yyyy_hh:mm:ss", date)
+    fun takeScreenShot(bitmap: Bitmap, name: String? = null): File? {
```

Comment:
This one is confusing as well since it's saving bitmap but not taking screen shot. 
Can we find a better naming?